web-extension/engine/bookmycomics.js

```diff
@@ -96,6 +92,27 @@ function BmcEngine(hostOrigin, readerName, comicInfo) {
             this.sendNotificationWithComicInfo('Comic Information', null);
         });
 
+    this._messaging.addWindowHandler(
```

Comment:
If you handle this message here, you might as well completely delete the background engine, and add all supported sources to every entrypoint (in the manifest).

The reason why we used the background page was to allow loading all support files while keeping the entrypoints shorter, no matter the number of supported readers.